libs/core/package.json

```diff
@@ -10,11 +10,14 @@
   "types": "src/index",
   "scripts": {
     "build": "tsc -b",
+    "tsc-watch": "tsc --watch",
```

Comment:
we can also watch incrementally with tsc -b from the root... maybe this is redundant